aptos-move/mvhashmap/src/lib.rs

```diff
@@ -69,6 +69,22 @@ impl<V> Entry<V> {
     }
 }
 
+pub(crate) struct VersionedPath<V> {
+    pub(crate) versioned_map: BTreeMap<TxnIndex, CachePadded<Entry<V>>>,
+    // Note: this can cache base (storage) value in Option<u128> to facilitate
+    // aggregator validation & reading in the future, if needed.
+    pub(crate) contains_delta: bool,
```

Comment:
it might be an aggregator, but we'd never set this unless we actually observed a delta.
also it's a versioned thingy, so it might have some deltas and some writes.